context/src/main/java/io/opentelemetry/context/LazyStorage.java

```diff
@@ -83,6 +84,10 @@ static ContextStorage get() {
 
   static ContextStorage createStorage(AtomicReference<Throwable> deferredStorageFailure) {
     String providerClassName = System.getProperty(CONTEXT_STORAGE_PROVIDER_PROPERTY, "");
+    // enforceDefault
+    if (ENFORCE_DEFAULT_VALUE.equals(providerClassName)) {
+      return DefaultContext.threadLocalStorage();
```

Comment:
How are we choosing between calling this method, vs. `ContextStorage.defaultStorage()` which return the same thing?

Reply:
yes. they return the same thing. what about calling `ContextStorage.defaultStorage()` consistently?

Reply:
@anuraaga I would really appreciate it If you can also share your opinion here.  : )

Reply:
Yeah let's remove `DefaultContext.threadLocalStorage`, that is a leftover from refactoring during the initial implementation. I was thinking of doing it after this PR if you weren't to do so :P

Reply:
@jkwatson Should we remove `DefaultContext.threadLocalStorage`  in this PR?

Reply:
If we don't, please create an issue for it, so we don't lose track.

Reply:
Let's remove it now. : )